### PR TITLE
[WIP] Fix bug which prevents lzdoom and gzdoom start scripts from reading files with spaces

### PR DESCRIPTION
**Pull request by travis134**
## Description

Fix bug which prevents lzdoom and gzdoom start scripts from reading files with spaces

Also fixes another bug where the code to grab the extension for these files is grabbing everything after the first (.) rather than the last (.).

I.e. it interprets the extension for "Doom II - Hell on Earth (v1.9).doom" as "9).doom" instead of "doom".

See https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html
> ${parameter##word}

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested Locally?
SCP'd the modified scripts to my device under /storage/roms/doom and ran them directly from ssh with a a few test "*.doom" files:

**The Ultimate Doom (mod).doom**
```
IWAD=/storage/roms/doom/iwads/doom/DOOM.WAD
MOD="/storage/roms/doom/mods/doom/ALIENS The Ultimate Doom Beta 9.2.pk3"
-- end --
```

**Doom II - Hell on Earth (v1.9).doom**
```
IWAD=/storage/roms/doom/iwads/doom2/DOOM2.WAD
-- end --
```

**Heretic - Shadow of the Serpent Riders (v1.3).doom**
```
IWAD=/storage/roms/doom/iwads/heretic/HERETIC.WAD
MOD=/storage/roms/doom/iwads/heretic/hocX.wad
-- end --
```

Verified they all start up correctly.

**Test Configuration**:
* Build OS name and version: JELOS
* Docker (Y/N): N/A (didn't do a docker build for this change, since it's just tweaking these scripts)
* JELOS Branch: dev

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: packages/games/emulators/raze/sources/start_raze.sh [hunk 1]
```diff
@@ -24,7 +24,7 @@ mkdir -p ${SAVE_DIR}
 params=" -config ${CONFIG} -savedir ${SAVE_DIR}"
 params+=" +gl_es 1 +vid_preferbackend 3 +cl_capfps 0 +cl_nomeleeblur 1 +vid_fps 1"
 
-EXT=${1#*.}
+EXT=${1##*.}
```
Comment (travis134) [inline packages/games/emulators/raze/sources/start_raze.sh:27]: Unrelated to this specific PR -- but same issue for build eng games so may as well fix. I did a search for any lines that matched this-- this was all of them.

File: packages/jelos/sources/scripts/runemu.sh [hunk 2]
```diff
@@ -454,15 +454,18 @@ fi
 clear_screen
 
 ### Restore the overclock mode
-OVERCLOCK=$(get_setting "system.overclock")
```
Comment (travis134) [inline packages/jelos/sources/scripts/runemu.sh:457]: This is another unrelated bug fix -- saw this come up a few times
```
# cat /var/log/es_launch_stderr.log
/usr/bin/runemu.sh: line 460: /usr/bin/overclock: No such file or directory
```

I think it's benign, since it happens on shutdown but still, best if we don't blow up

Comment (fewtarius) [reply packages/jelos/sources/scripts/runemu.sh:457]: This is important on the RG552, but this fix looks good.  ty :)
